fastmri/data/mri_data.py

```diff
@@ -358,3 +360,111 @@ def __getitem__(self, i: int):
             sample = self.transform(kspace, mask, target, attrs, fname.name, dataslice)
 
         return sample
+
+
+class AnnotatedSliceDataset(SliceDataset):
```

Comment:
Please provide an extensive docstring describing the parameters - see `SliceDataset` for an example.